contrib/ci-horizen/scripts/test/clang-tidy-launcher.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Get the current commit hash
+export COMMIT=`git log -1 --format="%H"`
+# Get the names of the files touched by this commit
+export FILELIST=`git diff-tree --no-commit-id --name-only -r $COMMIT`
+
+# Filter out all files which do not end with .cpp, .h and .cc
```

Comment:
Probably we should include also `.hpp` files, being the current extension of some headers.

Reply:
Yes, agree!